Projekty/Projekt1/Adrian_Stańdo__Mateusz_Stączek__Paweł_Wojciechowski/01_eda.ipynb

```diff
@@ -0,0 +1,1300 @@
+{
```

Comment:
<p>Warto zaznaczać zawsze czy chodzi bezwzględnie  czy udziałowo.</p><p> </p><p>Ciekawostka: przy mówieniu o gedner pay gap często nie uwzględnia się zawodów danych osób i porównuje się zarobki na przykład kobiety nauczycielki i mężczyzny inżyniera. Ostrzegam przed takimi rzeczami \:)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>